ubi_manifest/worker/tasks/content_audit.py

```diff
@@ -37,6 +42,7 @@ def content_audit_task() -> None:
     content is present, and that blacklisted content is absent.
     """
 
+    executor = Executors.thread_pool(max_workers=MAX_WORKERS)
```

Comment:
Please don't add `thread_pool`  if need this only for one function.
The `get_pkgs_from_all_modules` should be fast enough so it shouldn't block the code for much time.

If you want to use it, you need to properly shutdown the Executor, please refer to other code where it's used.